### DISCUSSION TITLE
Outside temperature not updated when AC not working

### DISCUSSION THREAD

**Discussion by mac73dev** [revk/xESP32-Faikout#296] [Q&A] [answered]
I am facing a problem where I am not sure whether it is a bug or a configuration issue:

The 'liquid' sensor is showing and updating the temperature values regardless if the AC is on or off (off meaning the AC has power but is not cooling or heating). So this works as expected.

The 'outside' sensor is only showing the correct temperature if the AC is in 'working' mode (e.g. cooling) but as soon as turned off the temperate value sticks with the last value. I would expect to see the right outside temperature even if the AC is not in working mode.

The stuck outside temperature value can be observed in the Faikin web interface as well as in Home Assistant so I do not expect any MQTT related issue.

Does somebody have the same issue?
Any help is much appreciated.

Faikin-S3-MINI-N4-R2: 91c1bc5 2024-03-31T10:59:15 S21
Daikin Comfora (FTXP35M/RXP35M)

![Faikin](https://github.com/revk/ESP32-Faikin/assets/22379258/1739133f-379b-4acf-a09f-6e455bcca4aa)

**Comment by revk** [comment]
Best to do the dump setting and see what is coming in over S21. If that shows it right then it is a Faikin but. If that is frozen then it is a Daikin "feature".

**Reply by mac73dev** [reply]
Thanks for your quick reply.
After watching the S21 messages for some time it looks to me like a Daikin "feature".
The outside temperature ("Sa" value) gets frozen as soon as the AC is not working anymore.
Weird enough that other values (such as "SI" and "SH") are kept updated.

**Reply by njh** [reply, marked as answer]
Yes, that is also my experience - stop seeing outside temperature S21 messages when the unit is switched off.

**Reply by MuzzaMcD** [reply]
Refer my previous discussion back in Feb where I raised this #220 
Before I installed the Faikin I had used a Daikin Controller (BRP072C42) and you could see an outside temp reading when the unit was in Mode:Off. 
When I took a closer look the unit only 'updated' this outside temp value every 60mins (when Mode:OFF). I only noticed this because I had previously observed that every hour (when the MODE was OFF) I would see a small spike in Power use for a few mins (maybe only 6-7 watts) every hour. It makes sense that the outside condenser unit would be checking the outside temperature regularly if only to deal with frost for example? Anyway I was going to do some 'snooping' to try and see if I could see any way to catch the hourly change but I got side tracked. I have another independent outside temp sensor so lost interest in following this up.
Just thought I should add this here in case it helps ...

**Reply by mac73dev** [reply]
Thanks for additional information.

**Reply by MuzzaMcD** [reply]
Refer my update in the discussion I raised previously, hope it assists.
https://github.com/revk/ESP32-Faikin/discussions/220#discussioncomment-9106711
